test/test.cpp

```diff
@@ -127,4 +128,49 @@ BOOST_AUTO_TEST_CASE( open_and_create ) {
    BOOST_REQUIRE_EQUAL( new_book.b, copy_new_book.b );
 }
 
+BOOST_AUTO_TEST_CASE( oom_flush_dirty_pages ) {
```

Comment:
Test should probably be #ifdef'ed for linux

Reply:
I have made the test work even if `oom` is not available. It doesn't test `oom` in that case, but does exercise chainbase some.